ethpm_types/ast.py

```diff
@@ -193,7 +192,7 @@ def get_nodes_at_line(self, line_numbers: "SourceLocation") -> list["ASTNode"]:
                 "`(lineno, col_offset, end_lineno, end_coloffset)`"
             )
 
-        if all(x == y for x, y in zip(self.line_numbers, line_numbers)):
+        if all(x == y for x, y in zip(self.line_numbers, line_numbers, strict=False)):
```

Comment:
Note to reviewer: analyze impact of `strict=False`

Reply:
`strict=False` is okay, also is the default behavior so remove it
```suggestion
        if all(x == y for x, y in zip(self.line_numbers, line_numbers)):
```